extensions/positron-r/src/r-installation.ts

```diff
@@ -338,3 +350,22 @@ function isExcludedInstallation(binpath: string): boolean | undefined {
 		excluded => isParentPath(binpath, excluded) || arePathsSame(binpath, excluded)
 	);
 }
+
+/**
+ * Get the default R interpreter path specified in the settings.
```

Comment:
```suggestion
 * Get the default R interpreter path specified in Positron settings.
```

Same rationale as above.